locale/fr/about/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: about.hbs
+title: A propos
+trademark: Trademark
+---
+# A propos de Node.js&reg;
+
+En tant qu'environnement d'exécution JavaScript asynchrone et orienté événnement, Node est conçu
+pour générer des applications évolutives. Dans le "hello world" d'exemple
+suivant, plusieurs connexions peuvent être gérées de manière concurrente.
+A chaque connexion, la fonction de rappel est déclenchée, mais si il n'y a rien à faire, Node restera en sommeil.                                                                   
+
+```javascript
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+});
+```
+
+Ceci contraste avec le modèle de concurrence plus commun dans lequel les processus sytème
+sont utilisés. La gestion réseau basée sur les processus est relativement
+inéfficace est diffidile à utiliser. De plus, les utilisateurs de Node n'ont pas à se soucier des problèmes d'interblocage des processus
+puis qu'il n'y a pas de verrouillage. Aucune fonction de Node ou presque
+n'effectue d'entrée/sortie, donc le processus ne bloque pas. Et comme rien
+n'est bloquant, développer un système scalable est relativement aisé avec Node.
+
+Si une partie des termes utilisés ne vous sont pas familliers, voici
+un article complet (en anglais) [Bloquant vs Non-Bloquant][].
+
+---
+
+Node est conçu de manière similaire à, et influencé par, des 
+librairies comme [Event Machine][] pour Ruby et [Twisted][] pour Python.
+Node pousse le modèle événnementiel encore plus loin. Il instaure la
+[boucle événnementielle][] en tant que construct de l'environnement d'exécution
```

Comment:
construct ?

Reply:
Precisely one of the terms that are easy in english but hard to translate.
Apart from "construction" or "concept", both of which are litteral translations and don't seem well suited for me, I don't see how to translate it better.

Maybe "composant élémentaire"?